defaults/main.yml

```diff
@@ -2,8 +2,8 @@
 
 # Basic settings
 postgresql_version: 9.3
-postgresql_encoding: 'UTF-8'
-postgresql_locale: 'en_US.UTF-8'
+postgresql_encoding: 'ISO-8859-1'
+postgresql_locale: 'sv_SE.ISO-8859-1'
```

Comment:
OK `en_US` was arbitrary already, but I doubt using `sv_SE` as a default improves it, does it?